lib/lookup.json

```diff
@@ -356,16 +361,20 @@
   "npm": {
     "maintainers": ["nodejs/npm"],
     "prefix": "v",
-    "skip": ["aix", "s390"]
+    "skip": ["aix", "s390", ">=17"],
+    "comment": "OpenSSL 3.0"
   },
   "path-to-regexp": {
     "prefix": "v",
     "maintainers": "blakeembrey",
-    "skip": ["aix"]
+    "skip": ["aix", ">=17"],
+    "comment": "OpenSSL 3.0"
   },
   "pino": {
     "prefix": "v",
-    "maintainers": "mcollina"
+    "maintainers": "mcollina",
+    "skip": true,
+    "comment": "https://github.com/pinojs/pino/issues/1215"
```

Comment:
How would you define "permanently" failing modules?

https://github.com/pinojs/pino/issues/1215 seems to be filed 5 days ago and be the first mention of such a failure? And it seems to be caused by a timeout?

Reply:
I would prefer this to be fixed.

Reply:
The point to skip until it is fixed. The more red CITGM is, the more difficult it is to detect real regressions.